### ISSUE TITLE
Expose the ability to overwrite Armor bulk values

### ISSUE THREAD

**Issue by TarkanAl-Kazily** [neilbenson/foundryvtt-whitehack-3e#5]
The default Whitehack rules say that non-magical armor's bulk is equal to the AC. It leaves the door open for magical armor to have custom weight though.

**Comment by neilbenson** [comment]
Yep, I may need to rewrite how encumbrance works. Perhaps have +1 and Special as checkboxes, then have the AC value as a number input.
